image_scraper.gemspec

```diff
@@ -4,12 +4,12 @@
 # -*- encoding: utf-8 -*-
 
 Gem::Specification.new do |s|
-  s.name = "image_scraper"
-  s.version = "0.1.8"
+  s.name = "rcarvalho-image_scraper"
```

Comment:
```suggestion
  s.name = "image_scraper"
```

Reply:
Just to keep the gem name the same. I'm not sure how to handle differences between yours `rcarvalho-image_scraper` and `image_scraper`. If you commit code changes and version/date setings separately, maybe there's a way I could cherry pick git commits. Suggestions welcome here on how to do it